lms/djangoapps/grades/models.py

```diff
@@ -636,3 +637,55 @@ def get_override(cls, user_id, usage_key):
 def prefetch(user, course_key):
     PersistentSubsectionGradeOverride.prefetch(user.id, course_key)
     VisibleBlocks.bulk_read(user.id, course_key)
+
+
+class PersistentSubsectionGradeOverrideHistory(models.Model):
+    """
+    A django model tracking persistent grades override audit records.
+    """
+    PROCTORING = 'PROCTORING'
+    GRADEBOOK = 'GRADEBOOK'
+    OVERRIDE_FEATURES = (
+        (PROCTORING, 'Proctoring'),
```

Comment:
nit: we usually use all lowercase strings for representing fixed choices in the database (like 'honor', 'verified', etc. for course modes).  I would change these strings and the strings in `OVERRIDE_ACTIONS` below.